downstream/modules/platform/proc-projects-using-collections-with-hub.adoc

```diff
@@ -0,0 +1,82 @@
+[id="proc-projects-using-collections-with-hub"]
+
+= Using Collections with {HubName}
+
+Before {ControllerName} can use {HubName} as the default source for collections content, you must create an API token in the {HubName} UI so that it could be specified in {ControllerName}. 
+
+Use the following procedure to connect to {PrivateHubName} or a public collection, the only difference is which URL you specify.
+
+.Procedure
+. Navigate to https://console.redhat.com/ansible/automation-hub/token 
```

Comment:
Missing period at end of the URL?

Reply:
the full stop then becomes part of the URL, and often causes problems.